code/_onclick/hud/diona_nymph.dm

```diff
@@ -0,0 +1,68 @@
+/* Diony Nymph HUD */
+/mob/living/simple_animal/diona/create_mob_hud()
+	if(client && !hud_used)
+		hud_used = new /datum/hud/diona(src)
+
+/datum/hud/diona/New()
+	..()
+	var/obj/screen/diona/blood = new /obj/screen/diona/blood()
+	var/obj/screen/diona/merge = new /obj/screen/diona/merge()
+	var/obj/screen/diona/evolve = new /obj/screen/diona/evolve()
+	var/obj/screen/diona/intent = new /obj/screen/diona/intent()
+	static_inventory += blood
+	static_inventory += merge
+	static_inventory += evolve
+	toggleable_inventory += intent
+
+/obj/screen/diona/intent
+	name = "Intent"
+	icon_state = "help"
+	icon = 'icons/mob/screen_simplemob.dmi'	
+	screen_loc = ui_acti
+
+/obj/screen/diona/intent/Click()
+	if(isnymph(usr))
+		var/mob/living/simple_animal/diona/N = usr
+		if(N.a_intent == "help") 
```

Comment:
should use the defines `INTENT_HELP` and `INTENT_HARM`

Reply:
Seconded